composer.json

```diff
@@ -8,13 +8,13 @@
 	"prefer-stable": true,
 	"minimum-stability": "dev",
 	"require": {
-		"composer/installers": "1.7.0",
+		"composer/installers": "1.9.0",
 		"php": ">=5.6|>=7.0",
-		"automattic/jetpack-autoloader": "^2.2.0"
+		"automattic/jetpack-autoloader": "2.2.0"
```

Comment:
Instead of pinning these versions, they should be a range. That will help ensure when the package is included in WooCommerce core that there's some flexibility in what actual version gets installed.